test/unit/range/view/view_to_lower_test.cpp

```diff
@@ -0,0 +1,77 @@
+// -----------------------------------------------------------------------------------------------------
+// Copyright (c) 2006-2019, Knut Reinert & Freie Universität Berlin
+// Copyright (c) 2016-2019, Knut Reinert & MPI für molekulare Genetik
+// This file may be used, modified and/or redistributed under the terms of the 3-clause BSD-License
+// shipped with this file and also available at: https://github.com/seqan/seqan3/blob/master/LICENSE
+// -----------------------------------------------------------------------------------------------------
+
+#include <iostream>
+
+#include <gtest/gtest.h>
+
+#include <seqan3/alphabet/nucleotide/dna5.hpp>
+#include <seqan3/range/concept.hpp>
+#include <seqan3/range/view/to_char.hpp>
+#include <seqan3/range/view/to_lower.hpp>
+#include <seqan3/std/ranges>
+
+using namespace seqan3;
+
+TEST(view_to_lower, basic)
+{
+
+    std::string input_string {"IAmADnaString"};
+    std::string cmp_string {"iamadnastring"};
+    std::string cmp2{"gnirtsandamai"};
+
+    std::vector<dna5> dna_vec {"AGGCGT"_dna5};
+    std::string cmp3{"aggcgt"};
+
+    std::vector<std::string> input_vec{"IAmADnaString", "IAmAProteinString"};
+    std::vector<std::string> cmp4{"iamadnastring", "iamaproteinstring"};
+
+    // pipe notation string
+    std::string s(input_string | view::to_lower);
+    EXPECT_EQ(cmp_string, s);
+
+    // function notation string
+    std::string s2(view::to_lower(input_string));
+    EXPECT_EQ(cmp_string, s2);
+
+    // combinability
+    std::string s3(input_string | view::to_lower | std::view::reverse);
+    EXPECT_EQ(cmp2, s3);
+
+    // combinability with NucleotideAlphabet
+    std::string s4(dna_vec | view::to_char | view::to_lower);
+    EXPECT_EQ(cmp3, s4);
+
+    // deep
+    std::vector<std::string> s5(input_vec | view::to_lower);
+    EXPECT_EQ(cmp4, s5);
+}
+
+TEST(view_to_lower, concepts)
+{
+    std::string input_string{"AEIOU"};
+    EXPECT_TRUE(std::ranges::InputRange<decltype(input_string)>);
+    EXPECT_TRUE(std::ranges::ForwardRange<decltype(input_string)>);
+    EXPECT_TRUE(std::ranges::BidirectionalRange<decltype(input_string)>);
+    EXPECT_TRUE(std::ranges::RandomAccessRange<decltype(input_string)>);
+    EXPECT_FALSE(std::ranges::View<decltype(input_string)>);
+    EXPECT_TRUE(std::ranges::SizedRange<decltype(input_string)>);
+    EXPECT_TRUE(std::ranges::CommonRange<decltype(input_string)>);
+    EXPECT_TRUE(const_iterable_concept<decltype(input_string)>);
+    EXPECT_TRUE((std::ranges::OutputRange<decltype(input_string), char>));
+
+    auto v1 = input_string | view::to_lower;
+    EXPECT_TRUE(std::ranges::InputRange<decltype(v1)>);
```

Comment:
I would rather do:
```cpp
EXPECT_EQ(std::ranges::InputRange<decltype(input_string)>, std::ranges::InputRange<decltype(input_string)>);
```
for properties that are preserved.
And
```cpp
EXPECT_EQ(std::ranges::InputRange<decltype(input_string)>, !std::ranges::InputRange<decltype(input_string)>);
```
(with `!`) for not preserved.

(Even if you might have copied this from somewhere else I think it is more clear).